pagopa/io-app
# fix: [IOBP-638] Add nested scroll enabled flag on the current payment webview modal

## Short description
This PR is aimed at solving a problem that persists only on some Android devices in which when entering the required OTP in the final payment step within the webview, upon tapping on the input field to enter the OTP code the keyboard would either not appear or when it did appear it was no longer clickable.

## List of changes proposed in this pull request
- Added the `nestedscrollEnabled` prop to the webview;

## How to test
To test this fix, start the as-is payment stream on a device or emulator that is an Android 10 or 11 in UAT environment and verify that when tapping on the form present in the OTP code entry, the keyboard works correctly
